node/src/Commands.ts

```diff
@@ -1466,6 +1466,18 @@ export function createTime(): command_request.Command {
     return createCommand(RequestType.Time, []);
 }
 
+/**
+ * @internal
+ */
+export function createPublish(
+    message: string,
```

Comment:
again, does the string support the non-utf8 ?

Reply:
same answer as before:)